src/main/java/spring/intro/controller/HelloController.java

```diff
@@ -0,0 +1,26 @@
+package spring.intro.controller;
```

Comment:
I suppose we've had to delete this class

Reply:
Done